src/transformers/generation/configuration_utils.py

```diff
@@ -324,10 +324,11 @@ class GenerationConfig(PushToHubMixin):
               reduce by 1. `num_assistant_tokens` value is persistent over multiple generation calls with the same assistant model.
             - `"heuristic_transient"`: Same as `"heuristic"` but `num_assistant_tokens` is reset to its initial value after each generation call.
             - `"constant"`: `num_assistant_tokens` stays unchanged during generation
-        assistant_confidence_threshold (`float`, *optional*, defaults to 0):
+        assistant_confidence_threshold (`float`, *optional*, defaults to None):
```

Comment:
```suggestion
        assistant_confidence_threshold (`float`, *optional*):
```
defaults to `None` is the default, which we prefer to leave as implicit :)